lib/options.js

```diff
@@ -69,6 +69,12 @@ Version ${ helpers.getVersion() }
       type: 'Boolean',
       description: 'AMD and CommonJS'
     },
+    {
+      option: 'es6Modular',
```

Comment:
please rename this option `esm` in order to be consistent to other bundlers like rollup. In future we could change `modular` to `umd`...